src/app/App.js

```diff
@@ -203,7 +202,19 @@ class App extends View {
     return language;
   }
 
-
+  async deleteLanguage(idToDelete) {
+    const confirm = window.prompt(`Are you sure you want to delete this Language? The data can be recovered at any time.`);
```

Comment:
```suggestion
    const confirmed = window.prompt(`Are you sure you want to delete this Language? The data can be recovered at any time. Type "YES" to delete.`);
```

There's a `window.confirm` property, so using `confirm` as the variable name here shadows that property, and might cause confusion down the road.

The user also doesn't know what they need to type based on the original message, so I added the last sentence to the prompt.